src/com/palmergames/bukkit/util/BukkitTools.java

```diff
@@ -278,4 +278,8 @@ public static boolean isSpigot() {
 		}
 
 	}
+	

```

Comment:
Should mention that this will always return a stripped down version number, removing any information of what bukkit version it is made for.

Reply:
I've added a javadoc for this